reference-architecture/osp-cli/ch4.8_copy_ssh_key.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+OCP3_KEY_FILE=${OCP3_KEY_FILE:-keys/ocp3_rsa}
+BASTION_HOST=${BASTION_HOST:-bastion.${OCP3_BASE_DOMAIN}}
+scp -i ${OCP3_KEY_FILE} ${OCP3_KEY_FILE} cloud-user@${BASTION_HOST}:.ssh/id_rsa
+
+ssh -i ${OCP3_KEY_FILE} cloud-user@${BASTION_HOST} chmod 600 .ssh/id_rsa
```

Comment:
Actually no.  ssh-copy-id copies a local public key to a remote `authorized_keys` file.  That allows login from the local host to the remote one.

In this case I'm copying the _private_ key so that the bastion can log into the other instances for Ansible.